recommendation_engine/flask_predict.py

```diff
@@ -76,5 +83,13 @@ def recommendation():
     return flask.jsonify(response_json), 200
 
 
+def log_it(func):
```

Comment:
Where is this decorator used? I do not see it being used

Reply:
future reference

Reply:
What future reference are you planning?